doc/040_backup.rst

```diff
@@ -204,6 +204,16 @@ Combined with ``--verbose``, you can see a list of changes:
     modified  /archive.tar.gz, saved in 0.140s (25.542 MiB added)
     Would be added to the repository: 25.551 MiB
 
+Estimator Scanner
```

Comment:
What about `Disabling Backup Progress Estimation`? I couldn't guess from the section title what it was about.